CBinding/Project/CProjectConfiguration.cs

```diff
@@ -120,15 +168,15 @@ public class CProjectConfiguration : ProjectConfiguration
 				
 				switch (target)
 				{
-				case CompileTarget.Bin:
+				case CompileTarget.Exe:
 					break;
-				case CompileTarget.StaticLibrary:
```

Comment:
This are backwards. A .NET "module" is more like a C/C++ static library, and a .NET "library" is a shared library.